x-pack/plugins/apm/public/components/app/RumDashboard/empty_state_loading.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiEmptyPrompt,
+  EuiLoadingSpinner,
+  EuiSpacer,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { Fragment } from 'react';
+
+export function EmptyStateLoading() {
```

Comment:
Should we use one of the loading components instead of a custom component? https://elastic.github.io/eui/#/display/loading

Reply:
it's the same component, it just adds an EuiEmptyPrompt and a text around it as well.

Reply:
i copied this from uptime, we have the same component there.